backend/src/main/java/quarano/department/web/TrackedCaseDiaryEntrySummary.java

```diff
@@ -0,0 +1,80 @@
+package quarano.department.web;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.support.MessageSourceAccessor;
+import org.springframework.hateoas.RepresentationModel;
+import quarano.core.web.MapperWrapper;
+import quarano.diary.DiaryEntry;
+import quarano.reference.SymptomDto;
+import quarano.tracking.ContactPerson;
+
+import java.time.LocalDateTime;
+import java.util.Locale;
+import java.util.Map;
+import java.util.stream.Stream;
+
+/**
+ * @author Marco Ebbinghaus
+ */
+public class TrackedCaseDiaryEntrySummary extends RepresentationModel<TrackedCaseDiaryEntrySummary> {
+
+	private final DiaryEntry entry;
+	private final MessageSourceAccessor messages;
+	private final MapperWrapper mapper;
+
+	public TrackedCaseDiaryEntrySummary(DiaryEntry diaryEntry, MessageSourceAccessor messages, MapperWrapper mapper) {
+		this.entry = diaryEntry;
+		this.messages = messages;
+		this.mapper = mapper;
+	}
```

Comment:
Why do you use an explicit constructor here and not `@RequiredArgsConstructor` like below at `ContactSummary`?